lib/config-generator.js

```diff
@@ -331,11 +331,31 @@ ConfigGeneraror.prototype = {
 
         return new Promise(function(resolve, reject) {
             var result = [];
-
+            var isRequire = false;
             jsstana.traverse(ast, function(node) {
-                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?))', node);
+                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?) (call ?.define ?) (call ?.define ? ?) (call ?.define ? ? ?) (call require ?) (call require ? ?) (call require ? ? ?) (call ?.require ?) (call ?.require ? ?) (call ?.require ? ? ?))', node);
```

Comment:
Yes @ipeychev, I didn't know the exact "lifecycle" of this process and how many times it would/could be executed. Thanks again for the help.